src/models_test.ts

```diff
@@ -546,7 +549,11 @@ describeMathCPU('loadModel from URL', () => {
       (fileBufferMap:
            {[filename: string]: Float32Array|Int32Array|ArrayBuffer}) => {
         spyOn(window, 'fetch').and.callFake((path: string) => {
-          return new Response(fileBufferMap[path]);
+          return new Promise((res, rej) => {
```

Comment:
Does it make sense to remove the curly brackets in this arrow function here, since it contains only one statement?

Reply:
Good point - I've removed them.